src/main/scala/elo/LeaderboardService.scala

```diff
@@ -0,0 +1,25 @@
+package elo
+
+class LeaderboardService(playerService: PlayerService) {
+
+  def getLeaderboard(numTopPlayers: Int = 3): Seq[Player] = {
```

Comment:
Nice...this will make it easier to later parametrize how many players on the leaderboard you want to see.